cmd/fyne/internal/commands/build.go

```diff
@@ -89,10 +99,10 @@ func (b *builder) build() error {
 		tags = append(tags, "release")
 	}
 	if len(tags) > 0 {
-		args = append(args, "-tags", strings.Join(tags, ","))
+		args = append(args, "--tags", strings.Join(tags, ","))
```

Comment:
I think the actual Go syntax for flags is to prefer one dash over two. I think we might want to keep it as it was.

Reply:
Agreed, I think there are 7 instances, many are new - they should all be "-tags" iirc